<html>
<head>
<style>
table {
  font-family: arial, sans-serif;
  border-collapse: collapse;
  width: 100%;
}

th {
  background-color: #dddddd;
  border: 1px solid #030303;
  text-align: left;
  padding: 8px;
}
    td {
      font-weight: bold;
      border: 1px solid #030303;
      text-align: left;
      padding: 8px;
      white-space: pre;
      font-family: "Courier New", Courier, monospace
    }
h1 {margin-top: 0em; margin-bottom: 0em;
  font-family: arial, sans-serif;
  width: 100%;
  font-size: 300%;
  background-color: #FFFF00;
}
h2 {margin-top: 0em; margin-bottom: 0em;
  font-family: arial, sans-serif;
  width: 100%;
  font-size: 200%;
  background-color: #FFFF00;
}
h3 {margin-top: 0em; margin-bottom: 0em;
  font-family: arial, sans-serif;
  width: 100%;
  font-size: 150%;
  background-color: #33F0FF;
}
h4 {margin-top: 0em; margin-bottom: 0em;
  font-family: arial, sans-serif;
  width: 100%;
  font-size: 125%;
  background-color: #7DFF33;
}
</style>
</head>
<body>
<h2>SCENARIO 17: Unhandled exception [Category Set: Errors]</h2>
<h3>INPUTS</h3>
<h4>GROUP 1: Log</h4>
<table>
<tr><th>#</th><th>Line</th></tr>
<tr><td>1</td><td></td></tr>
</table>
<h4></h4>
<h4>GROUP 2: Scalars</h4>
<table>
<tr><th>#</th><th>Delimiter</th><th>Column#</th><th>Throw Error</th></tr>
<tr><td>1</td><td>,</td><td>1</td><td>Y</td></tr>
</table>
<h4></h4>
<h4>GROUP 3: Lines</h4>
<table>
<tr><th>#</th><th>Line</th></tr>
<tr><td>1</td><td>col_0,col_1,col_2</td></tr>
<tr><td>2</td><td>val_0,val_11,val_2</td></tr>
<tr><td>3</td><td>val_0,val_10,val_2</td></tr>
<tr><td>4</td><td>val_0,val_11,val_2</td></tr>
</table>
<h4></h4>
<h3>OUTPUTS</h3>
<h4>GROUP 1: Log</h4>
<table>
<tr><th>#</th><th>#Lines</th><th>Date Offset</th><th>Text</th></tr>
<tr><td>1</td><td>2</td><td>IN [0,2000]: </td><td>LIKE /.*: File .*ut_group.csv, delimiter ',', column 1/: </td></tr>
<tr style="background-color: #E62A3E"><td>1*</td><td>*NO RECORD*</td><td>NOT IN [0,2000]: </td><td>NOT LIKE /.*: File .*ut_group.csv, delimiter ',', column 1/: </td></tr>
</table>
<h4>1 failed of 1: FAILURE</h4>
<h4>GROUP 2: listAsIs</h4>
<table>
<tr><th>#</th><th>#Instances</th></tr>
<tr><td>1</td><td>2</td></tr>
<tr style="background-color: #E62A3E"><td>1*</td><td>*NO RECORD*</td></tr>
</table>
<h4>1 failed of 1: FAILURE</h4>
<h4>GROUP 3: sortByKey</h4>
<table>
<tr><th>#</th><th>Key</th><th>Value</th></tr>
<tr><td>1</td><td>val_10</td><td>1</td></tr>
<tr style="background-color: #E62A3E"><td>1*</td><td>*NO RECORD*</td><td></td></tr>
<tr><td>2</td><td>val_11</td><td>2</td></tr>
<tr style="background-color: #E62A3E"><td>2*</td><td>*NO RECORD*</td><td></td></tr>
</table>
<h4>2 failed of 2: FAILURE</h4>
<h4>GROUP 4: sortByValue</h4>
<table>
<tr><th>#</th><th>Key</th><th>Value</th></tr>
<tr><td>1</td><td>val_10</td><td>1</td></tr>
<tr style="background-color: #E62A3E"><td>1*</td><td>*NO RECORD*</td><td></td></tr>
<tr><td>2</td><td>val_11</td><td>2</td></tr>
<tr style="background-color: #E62A3E"><td>2*</td><td>*NO RECORD*</td><td></td></tr>
</table>
<h4>2 failed of 2: FAILURE</h4>
<h4>GROUP 5: Unhandled Exception</h4>
<table>
<tr><th>#</th><th>Message</th><th>Stack</th></tr>
<tr><td>1</td><td>*NO RECORD*</td><td></td></tr>
<tr style="background-color: #E62A3E"><td>1*</td><td>Error thrown</td><td>Error: Error thrown
    at purelyWrapUnit (C:\Users\Brend\OneDrive\Documents\GitHub\trapit_nodejs_tester\examples\colgroup\test-colgroup.js:72:11)
    at callPWU (C:\Users\Brend\OneDrive\Documents\node_modules\trapit\lib\trapit-utils.js:81:14)
    at Object.testUnit (C:\Users\Brend\OneDrive\Documents\node_modules\trapit\lib\trapit-utils.js:201:62)
    at Object.fmtTestUnit (C:\Users\Brend\OneDrive\Documents\node_modules\trapit\lib\trapit-utils.js:212:20)
    at Object.<anonymous> (C:\Users\Brend\OneDrive\Documents\GitHub\trapit_nodejs_tester\examples\colgroup\test-colgroup.js:93:8)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)</td></tr>
</table>
<h4>1 failed of 1: FAILURE</h4>
<h2>5 failed of 5: FAILURE</h2>
</body></html>